tjreports/administrator/sql/updates/mysql/2.0.0.sql

```diff
@@ -0,0 +1,13 @@
+-- Adding default values to the column of existing table
+
+ALTER TABLE `#__tj_reports`	CHANGE `asset_id` `asset_id` int(11) NOT NULL DEFAULT 0;
```

Comment:
use spaces for between words instead of tabs

Reply:
done